### PR TITLE
feat(data-cruncher): port queries to the data cruncher from RUM Explorer

### PR DESCRIPTION
**Pull request by ekremney**
This PR ports the existing queries to the `data-cruncher` from https://github.com/adobe/helix-website/

Main idea behind the port is to use the same definitions for the metrics and concepts.

So far, this PR contains the ports for the following queries:
1. CWV
2. Experiment
3. High Inorganic High Bounce Rate

more ports will follow...

---

This PR uses a copied version of the `cruncher.js` from https://github.com/adobe/helix-website/ library. The plan is to extract the `cruncher.js` into a shared library soon, so that the way raw RUM data crunched is the same in all places.

Thanks @karlpauls for the contributions! 💯

### CODE REVIEW

File: packages/spacecat-shared-rum-api-client/src/common/cruncher.js
Comment (ramboz) [inline packages/spacecat-shared-rum-api-client/src/common/cruncher.js:1]: I'd be tempted to go one step further and extract each class to individual files

**Review by ramboz — APPROVED** [review]
Looks good to me

File: packages/spacecat-shared-rum-api-client/.eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+src/common/cruncher.js
```
Comment (solaris007) [inline packages/spacecat-shared-rum-api-client/.eslintignore:1]: why are we excluding this?

File: packages/spacecat-shared-rum-api-client/.nycrc.json [hunk 2]
```diff
@@ -10,5 +10,8 @@
   "all": true,
   "include": [
     "src/**/*.js"
+  ],
+  "exclude": [
```
Comment (solaris007) [inline packages/spacecat-shared-rum-api-client/.nycrc.json:14]: why are we excluding this?

File: packages/spacecat-shared-rum-api-client/src/common/cruncher.js
Comment (ekremney) [reply packages/spacecat-shared-rum-api-client/src/common/cruncher.js:1]: done 👍

File: packages/spacecat-shared-rum-api-client/.eslintignore (hunk 1 above)
Comment (ekremney) [reply packages/spacecat-shared-rum-api-client/.eslintignore]: no more exclusion. we are inclusive now ❤️

File: packages/spacecat-shared-rum-api-client/.nycrc.json (hunk 2 above)
Comment (ekremney) [reply packages/spacecat-shared-rum-api-client/.nycrc.json]: no more exclusion. we are inclusive now ❤️

**Pull request merged**